### ISSUE TITLE
Preview transaction before broadcasting

### ISSUE THREAD

**Issue by lviggiano** [LedgerHQ/ledger-live-desktop#2470]
- [x] I have checked this feature was not yet requested.

I would like to see the (singed/unsigned) transaction before broadcasting, so I can check that the change address is also correct, for additional security/verification.

#### Part of the application

mobile and desktop app (sending/advanced options).

You can define an "advanced/experimental feature" option to enable it, so that the user can have an expanding section to show the transaction details.

**Comment by lviggiano** [comment]
Recently a [vulnerability](https://medium.com/shiftcrypto/a-ransom-attack-on-coldcards-change-and-keypath-verification-f3c71461624a) has been discovered on CoolCard on the change address. This has nothing to do with Ledger, but checking would be my routine before broadcasting.

I just wanted to point out that this can actually be useful.

**Comment by gre** [comment]
I agree this feature would be useful. We have discussed it internally and consider it. Can't give yet an ETA tho.
